pkg/apis/core/types.go

```diff
@@ -4894,7 +4894,9 @@ type NodeDaemonEndpoints struct {
 	KubeletEndpoint DaemonEndpoint
 }
 
-// NodeRuntimeHandlerFeatures is a set of runtime features.
+// NodeRuntimeHandlerFeatures is a set of features implemented by the runtime handler.
+// The features contained in the NodeRuntimeHandlerFeatures should depend on the runtime handlers,
+// (i.e. dependent to the information exposed by the OCI runtime-spec's features).
```

Comment:
I would suggest dropping this  line because it is really dependent on the lower level runtime called by the runtime handler or clarifying it instead. For e.g. runc vs. crun vs. kata implementing a feature when configured as the runtime handler.

Reply:
Thanks, it makes sense. Please note that I intended the word "_features_" here to indicate OCI runtime-spec's [Feature Structure](https://github.com/opencontainers/runtime-spec/blob/main/features.md) which we can expect each low-level runtime to provide.

Fixed in 
- K8s API: 5d75660dc11ff443ebab2551aed8e56a54cc218d
- CRI: 38e4c6b01666128216290e1579a8f2d408a60e66